CITATION.cff

```diff
@@ -0,0 +1,25 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: pixi
+message: >-
+  If pixi contributes to a project that leads to a
+  scientific publication, please acknowledge this fact by
+  citing according to this file.
+type: software
+authors:
+  - given-names: pixi open-source contributors
+repository-code: 'https://github.com/prefix-dev/pixi'
```

Comment:
As software citation should [enable specificity and accessibility](https://matthewfeickert-talks.github.io/talk-chep-2023/#4) I'd suggest adding in the release number information to the URL here. So something like

```yaml
repository-code: "https://github.com/prefix-dev/pixi/releases/tag/v0.15.2"
```